daemon/api.go

```diff
@@ -239,13 +239,14 @@ func sysInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 	}
 
 	m := map[string]interface{}{
-		"series":     release.Series,
-		"version":    c.d.Version,
-		"os-release": release.ReleaseInfo,
-		"on-classic": release.OnClassic,
-		"managed":    len(users) > 0,
-
-		"kernel-version": release.KernelVersion(),
+		"series":             release.Series,
+		"version":            c.d.Version,
+		"os-release":         release.ReleaseInfo,
+		"on-classic":         release.OnClassic,
+		"managed":            len(users) > 0,
+		"mount-directory":    dirs.SnapMountDir,
+		"binaries-directory": dirs.SnapBinariesDir,
+		"kernel-version":     release.KernelVersion(),
```

Comment:
Here is a strawman:
```
"locations": {
        "snap-mount-dir": ...
        "snap-bin-dir": ...
},
```
The "bin" vs. Binaries inconsistency might be fixed on the code side. "bin" is more friendly for /bin.